alembic/versions/2023_11_08_b105b426af99_rename_family_in_db.py

```diff
@@ -0,0 +1,67 @@
+"""Rename family in db
+
+Revision ID: b105b426af99
+Revises: 70e641d723b3
+Create Date: 2023-11-08 15:13:25.197537
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "b105b426af99"
+down_revision = "70e641d723b3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Rename tables
+    op.rename_table("family", "case")
+    op.rename_table("family_sample", "case_sample")
+
+    # Drop foreign key constraints that reference the unique constraint to be dropped
+    op.drop_constraint("analysis_ibfk_1", "analysis", type_="foreignkey")
+    op.drop_constraint("case_sample_ibfk_1", "case_sample", type_="foreignkey")
```

Comment:
should this be the other way around? Or does op.rename_table also alter the foreignkeys, that you then drop?

Reply:
Yeah, I think it makes more sense to drop the constraints and then rename the table 👍